cluster-autoscaler/provisioningrequests/apis/provisioning.x-k8s.io/v1beta1/register.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	// GroupName represents the group name for ProvisioningRequest resources.
-	GroupName = "provisioning.k8s.io"
+	GroupName = "provisioning.x-k8s.io"
 	// GroupVersion represents the group name for ProvisioningRequest resources.
 	GroupVersion = "v1beta1"
```

Comment:
I'd still recommend you start with an alpha to avoid committing yourself to forward compatibility without any feedback

Reply:
We want to start integration with Kueue soon, so the commitment to the API is happening right now. We also ran the API on GKE as a part of early access program to some features and the customers were OK with it.